README.md

```diff
@@ -1,14 +1,35 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
need to change this by [generating our own workflow badge](https://dev.to/azure/github-how-to-display-the-status-badge-for-a-github-action-5449). not sure if it's only for Java CI only or for all workflow though..